src/components/radio-button/radio-button-group.component.tsx

```diff
@@ -52,6 +52,8 @@ export interface RadioButtonGroupProps extends ValidationProps, MarginProps {
   value?: string;
   /** Overrides the default tooltip position */
   tooltipPosition?: "top" | "bottom" | "left" | "right";
+  /** Set validation styling to large */
+  largeStyleValidation?: boolean;
```

Comment:
**suggestion**(non-blocking): Again, similar sort of comment here, if we do have to go with this approach. Could we use some sort of type approach with size types, instead of boolean. Something like `validationSize` etc. with `"medium"` | `"large"` as types? This would also be future-proof, as if the DS add a small variant in the future, or an even larger variant we'd have scalability to add more types without introducing a breaking change by moving away from the boolean approach, or adding more props 👍

Reply:
Decided that we're doing this in a separate piece of work so I'll resolve this.